src/Modal/marketing-modal.mdx

```diff
@@ -24,8 +24,9 @@ The ``MarketingModal`` is a preconfigured `ModalDialog` that accepts an image an
   const [isOpen, open, close] = useToggle(false);
   return (
     <>
-      <Button variant="outline-primary" onClick={open}>Open marketing modal</Button>
-
+      <div className="d-flex">
```

Comment:
[clarification] Why is it necessary to wrap this element with `div.d-flex`?

Reply:
[We can avoid it. d-flex needed only to move the button to the right side in RTL direction. The screen without d-flex](https://github.com/openedx/paragon/pull/1084#discussion_r810018519)